src/syntax/parser.rs

```diff
@@ -376,7 +377,7 @@ impl<TokType: Tokenizer + Iterator<Item=Tokens>> Parser<TokType> {
 
         let expr = Expr::FnDecl(fn_name, args, return_type, definition);
 
-        return Some(ExprWrapper::default(expr));
+        Some(ExprWrapper::default(expr))
```

Comment:
I don't feel strongly enough about it. just we should be consistent in whatever way we do